web_development_101/the_front_end/project_html_css.md

```diff
@@ -3301,6 +3301,7 @@ Submit a pull request with a link to your solution in this [file](https://github
 * [FrMazzo's Solution (Easy)](https://github.com/FrMazzo/google-homepage) - [View in Browser](https://frmazzo.github.io/google-homepage/)
 * [pabloitzeder's Solution (Easy)](https://github.com/pabloitzeder/google-homepage) - [View in Browser](https://pabloitzeder.github.io/google-homepage/)
 * [Maheswar's Solution (Easy)](https://github.com/mahesh12393/google-homepage) - [View in Browser](https://mahesh12393.github.io/google-homepage/)
+* [Andrew S's Solution (Easy)](git@github.com:apsip/google-homepage.git) 
```

Comment:
The link to your solution currently contains the SSH link to the repository, rendering it inaccessible as a hyperlink. It should instead be the HTTPS link to the repository. Make sure your link is consistent with other solutions.

Reply:
Ah I see, I'll use this instead: "https://github.com/apsip/google-homepage.git"